packages/components/src/core/styles/common/defaultTheme.ts

```diff
@@ -557,4 +633,5 @@ export interface Borders {
   warning: Border;
 }
 
-export const defaultTheme = createThemeAdaptor(defaultThemeOptions);
+export const defaultTheme = (theme: "light" | "dark") =>
```

Comment:
Comment explaining the setup. This means any `defaultTheme` usage is now a function, not an object. Has consequences for developer usage.